src/main/java/ru/hogwarts/school/controller/FacultyController.java

```diff
@@ -0,0 +1,75 @@
+package ru.hogwarts.school.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ru.hogwarts.school.model.Faculty;
+import ru.hogwarts.school.model.Student;
+import ru.hogwarts.school.service.FacultyService;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/faculty")
+public class FacultyController {
+
+    private final FacultyService facultyService;
+
+    public FacultyController(FacultyService facultyService) {
+        this.facultyService = facultyService;
+    }
+
+    @GetMapping("/findById/{id}")
```

Comment:
Рекомендую повторить тему HTTP и REST. В путях указывается ресурс, с которым нужно совершить действие. Само действие описывается методами GET/POST/PUT/DELETE
```suggestion
    @GetMapping("/{id}")
```